src/components/Switch/SwitchGroup.stories.js

```diff
@@ -33,3 +34,24 @@ Primary.args = {
   legend: 'Environment mode',
   srOnlyLegend: true
 };
+
+const withIconsModel = 'map';
+
+const WithIconsTemplate = (args, { argTypes }) => ({
+  props: Object.keys(argTypes),
+  components: { SwitchGroup, SwitchItem, Earth, TableIcon },
+  data: () => ({ withIconsModel }),
+  setup: () => ({ args }),
+  template: `
+    <switch-group v-bind="args">
+      <switch-item name='chart' label="Map view" value='map' v-model="withIconsModel" sr-only-label><earth class="w-6 h-6" /></switch-item>
```

Comment:
Nits: inconsistencies between single/double quotes, a little hard to read when it's all on one line